recipe/run_conda_forge_build_setup_win.bat

```diff
@@ -103,6 +103,14 @@ if not "%CUDA_VERSION%" == "None" (
 )
 :: /CUDA
 
+conda.exe info --json | shyaml get-value platform > build_platform.txt
+set /p BUILD_PLATFORM=<build_platform.txt
+del build_platform.txt
+
+cat .ci_support\%CONFIG%.yaml | shyaml get-value target_platform.0 %BUILD_PLATFORM% > target_platform.txt
+set /p TARGET_PLATFORM=<target_platform.txt
+del target_platform.txt
```

Comment:
I don't know batch scripting too well, but I couldn't find a better way to "store things in a batch variable".

Is it true that here you are this difficulty in storing things in a file, before reading in the file?

Reply:
Yes, there's another syntax to do this, but it's really unreadable. This technique is used in the same file just  few lines above.